examples/demo/src/layout/SubMenu.tsx

```diff
@@ -68,14 +68,17 @@ const SubMenu: FC<Props> = ({
                 </Tooltip>
             )}
             <Collapse in={isOpen} timeout="auto" unmountOnExit>
+                {/* @rattrayalex: not intended, should fix. */}
                 <List
                     dense={dense}
                     component="div"
                     disablePadding
                     className={
-                        sidebarIsOpen
-                            ? classes.sidebarIsOpen
-                            : classes.sidebarIsClosed
+                        sidebarIsOpen ? (
+                            classes.sidebarIsOpen
+                        ) : (
+                            classes.sidebarIsClosed
+                        )
```

Comment:
I consider this a regression I'd like to fix (jsx styling in jsx attributes)

Reply:
Yeah, looks like a bug.